packages/integration-tests/src/helpers/admin-tenant.ts

```diff
@@ -37,16 +39,29 @@ export const createUserWithAllRoles = async () => {
   // Should have roles for default tenant Management API and admin tenant Me API
   const roles = await api.get('roles').json<Role[]>();
   await Promise.all(
-    roles.map(async ({ id }) =>
-      api.post(`roles/${id}/users`, {
-        json: { userIds: [user.id] },
-      })
-    )
+    roles
+      .filter(({ name }) => roleNames.includes(name))
+      .map(async ({ id }) =>
+        api.post(`roles/${id}/users`, {
+          json: { userIds: [user.id] },
+        })
+      )
   );
 
   return [user, { username, password }] as const;
 };
 
+export const createUserWithAllRoles = async () => {
+  const allRoles = await api.get('roles').json<Role[]>();
+  const allRoleNames = allRoles.map(({ name }) => name);
+  return createUserWithSpecifiedRoleNames(allRoleNames);
+};
+
+export const createUserWithUserRole = async () => createUserWithSpecifiedRoleNames(['user']);
+
+export const createUserWithAdminRole = async () =>
+  createUserWithSpecifiedRoleNames(['user', 'admin:admin']);
```

Comment:
no need for these two functions? to avoid additional layers

Reply:
scopes here also have enum/builder function.

e.g.:
`packages/schemas/src/types/user.ts`: `AdminTenantRole.User`